### PR TITLE
SOC-634 - GlobalWatchlistTask skip empty revisions

### PR DESCRIPTION
**Pull request by kvas-damian**
https://wikia-inc.atlassian.net/browse/SOC-634

### CODE REVIEW

File: extensions/wikia/GlobalWatchlist/GlobalWatchlistTask.class.php [hunk 1]
```diff
@@ -43,25 +43,29 @@ public function addWatchers( $databaseKey, $nameSpace, array $watchers ) {
...
-			$globalWatchlistBot = new GlobalWatchlistBot();
 
-			$db = wfGetDB( DB_MASTER, [], \F::app()->wg->ExternalDatawareDB );
-			foreach ( $watchers as $watcherID ) {
-				if ( $globalWatchlistBot->shouldNotSendDigest( $watcherID ) ) {
-					$this->clearGlobalWatchlistAll( $watcherID );
-					continue;
-				}
+			// Skip revisions that doesn't exist
+			if ( !empty($revision) ) {
```
Comment (bkoval) [inline extensions/wikia/GlobalWatchlist/GlobalWatchlistTask.class.php]: I guess we could add two spaces (before and after $revision)

**Comment by bkoval** [comment]
LGTM except whitespace comment.

**Comment by bkoval** [comment]
LGTM :dancers:

**Comment by rafalkalinski** [comment]
LGTM

**Pull request merged**